test/lexer/template.ts

```diff
@@ -106,10 +106,10 @@ describe('Lexer - Template', () => {
   });
 
   describe('Lexer - Tagged Template', () => {
-    const tokens: [Context, Token, string, string | void][] = [
+    const tokens: [Context, Token, string, string | null][] = [
       //[Context.TaggedTemplate, Token.TemplateSpan, '`\\u{70bc`', undefined],
-      [Context.TaggedTemplate, Token.TemplateContinuation, '`\\7${', undefined],
-      [Context.TaggedTemplate, Token.TemplateContinuation, '`\\1${', undefined],
+      [Context.TaggedTemplate, Token.TemplateContinuation, '`\\7${', null],
```

Comment:
Is the `tokenValue` expected to be null?

Reply:
I guess so. Previously not strict assertion, so undefined == null I assume.